src/explore-education-statistics-frontend/src/modules/table-tool/components/TableHeadersForm.tsx

```diff
@@ -30,42 +36,183 @@ const TableHeadersForm = (props: Props) => {
       <Formik<FormValues>
         enableReinitialize
         initialValues={initialValues}
+        validationSchema={Yup.object<FormValues>({
+          rowGroups: Yup.array()
+            .of(
+              Yup.array()
+                .of<SortableOption>(Yup.object())
+                .ensure(),
+            )
+            .min(1, 'Must have at least one row group'),
```

Comment:
Do these need checking over with Fred?

Reply:
Think we should just review the error messages later on as the content shouldn't really block this getting in.